app/assets/javascripts/player.js

```diff
@@ -0,0 +1,7 @@
+$(document).ready(function(){
```

Comment:
all files in `app/assetss/javascript` are merged into one file and executed on all pages, so you have dupliaction of code in `app/assets/javascripts/event.js` and `app/assets/javascripts/player.js` files